src/crafting.cpp

```diff
@@ -1102,14 +1102,45 @@ comp_selection<item_comp> player::select_item_component( const std::vector<item_
         uimenu cmenu;
         // Populate options with the names of the items
         for( auto &map_ha : map_has ) {
-            std::string tmpStr = item::nname( map_ha.type ) + _( " (nearby)" );
+            std::string tmpStr;
+            if( item::count_by_charges( map_ha.type ) ) {
+                tmpStr =  string_format( _( "%s (%d charges nearby)" ),
+                                        item::nname( map_ha.type ).c_str(),
+                                        map_inv.charges_of( map_ha.type ));
+            }
+            else {
```

Comment:
Style issue, see doc/CODE_STYLE.txt, in short: it should be `} else {` in one line. Those style issues caused the build to fail.